pubspec.yaml

```diff
@@ -28,8 +28,10 @@ dev_dependencies:
   dart_dev: ^2.0.1
   dart_style: ^1.2.0
   dependency_validator: ^1.2.3
+  meta: ^1.1.7
   mockito: ^4.0.0
   pedantic: ^1.4.0
+  react: ^4.6.1
```

Comment:
I'm pretty new to this repo - so I'll admit I don't fully understand how dependencies relate to the expected dependencies of the repos at the time of the codemod execution, but since this codemod is targeting stuff that is within the `5.1.0-wip` branch of react, do we need a dependency override for that?

Reply:
That's a really good question. I don't believe the version of react comes into play because the analyzer only needs to be included for [testing within the base branch](https://github.com/Workiva/over_react_codemod/blob/CPLAT-5212_component_2_migration/class_annotation/test/component2_suggestors/class_name_and_annotation_migrator_test.dart#L127) (from what I can tell). From what I can tell, the analyzer could see the import statement outside of `lib` but react wasn't a dependency, so it was failing and just needed something in the `pubspec`. If you'd rather my override it just to be sure though, I'm happy to do that @aaronlademann-wf!